source/fuzz/fuzzer_util.cpp

```diff
@@ -687,6 +691,44 @@ std::vector<opt::Instruction*> GetParameters(opt::IRContext* ir_context,
   return result;
 }
 
+void AddFunctionType(opt::IRContext* ir_context, uint32_t result_id,
+                     const std::vector<uint32_t>& type_ids) {
+  assert(result_id != 0 && "Result id can't be 0");
+  assert(!type_ids.empty() &&
+         "OpTypeFunction always has at least one operand - function's return "
+         "type");
+  assert(IsNonFunctionTypeId(ir_context, type_ids[0]) &&
+         "Return type must not be a function");
+
+  for (size_t i = 1, n = type_ids.size(); i < n; ++i) {
```

Comment:
Unless code is performance-critical I prefer to avoid the optimization of introducing an additional variable to capture the result of a function call.  "Premature optimization is the root of all evil" and all that.

Reply:
Fixed in 93b8e3a.